plugins/importexport/native/filter/NativeXmlArticleFilter.inc.php

```diff
@@ -49,18 +49,7 @@ function getPublishedSubmissionInsertMethod() {
 	 * @param $node DOMElement
 	 */
 	function handleElement($node) {
```

Comment:
I think this function can now be removed entirely.

Reply:
Fixed.